src/scene_battle.cpp

```diff
@@ -623,3 +638,8 @@ void Scene_Battle::CallDebug() {
 	}
 }
 
+void Scene_Battle::SelectionFlash(Game_Battler* battler) {
+	if (battler) {
+		battler->Flash(31, 31, 31, 24, 16);
```

Comment:
I do not like this "let's think like RPG_RT to do some math, so we do not need to do other math" here.
It is totally not obvious that this is meaning white color.

Reply:
The idea is to streamline one API so animated flash for map events, the screen, battle sprites, battle transition effect, etc... all work exactly the same way. Similar for shake. This guarantees the pixel colors will be a perfect match one time and then the logic can be reused.

Later we could build a better flash API which works directly with `Color` and goes to and from RPG_RT format. But I don't want to tackle that now.

Reply:
In other words, having a better unified flash API is blocked until #1954 is resolved. I'd like to punt on improving this for now, if that's ok.